Modules/SubModule_Import/xASL_imp_ReadSourceData.m

```diff
@@ -99,3 +118,19 @@ function xASL_imp_ReadSourceData_CheckFolderHierarchy(x)
 
 
 
+%% Unzip all '.zip' files in subdirectories and delete original files
+function xASL_imp_UnzipAll(dirToUnzip)
+
+    % Iterate over zip file
+    zipFiles = xASL_adm_GetFileList(dirToUnzip,'^.+\.zip$','FPListRec');
+    for iFile = 1:numel(zipFiles)
+        currentDirectory = xASL_fileparts(zipFiles{iFile});
+        unzip(zipFiles{iFile},currentDirectory);
```

Comment:
Preferably, see if we can use our own function that is faster and/or doesn't require jvm

Reply:
For me `xASL_adm_UnzipOrCopy` crashes and says there are too many output files (line 64). In addition it internally uses the exact same command...

```matlab
if strcmpi(ext,'.gz')
	destpath = fullfile(destDir, name );
	if bOverwrite || ~exist(destpath,'file')
		X = gunzip(srcpath, destDir);
	end
elseif strcmpi(ext,'.zip')
	destpath = fullfile(destDir, name );
	if bOverwrite || ~exist(destpath,'file')
		X = unzip(srcpath, destDir);
	end
else
	destpath = fullfile(destDir, entries(ii).name);
	if bOverwrite || ~exist(destpath,'file')
		copyfile(srcpath, destDir);
		X = destpath;
	end
end
```

... so no real gained benefit for me.